### PR TITLE
Marketing

### LINKED ISSUE #3: Crear parte de marketin de la pagina
### Explanation
Crear frontn de landing juniors parte central de la pagina en esta parte esta la parte de marketing de la  pagina

### Screeshots
image

### PR DESCRIPTION
**Pull request by Jesus-Gonzalez-Arroyo**
Add pages marketing 

fixes #3

### CODE REVIEW

**Review by elias-miguel-vergara — COMMENTED** [review]
revision

**Comment by Jesus-Gonzalez-Arroyo** [comment]
Revisado y completado, sugerir cambios

**Review by Israel-Laguan — APPROVED** [review]

File: components/Marketing.jsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
+import Image from "next/image";
+import React from "react";
+import { FaStar } from "react-icons/fa";
+import FotoPerfil1 from "../public/Persona-4.png";
+import FotoPerfil2 from "../public/persona-3.png";
+import FotoPerfil3 from "../public/persona-2.png";
+import FotoPerfil4 from "../public/Persona-1.png";
+import Mancha1 from "../public/Trazado 2.png";
+import Mancha2 from "../public/Trazado 3.png";
```
Comment (Israel-Laguan) [inline components/Marketing.jsx]: Try to use only English for names

**Review by Israel-Laguan — APPROVED** [review]

File: components/Confirmacion.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        <div className={Style.Confirm}>
+            <p className={Style.FirstTitle}>¡Muchas gracias por adquirir nuestros servicios!</p>
+            <p className={Style.LastTitle}>Se le ha enviado un correo a su e-mail para su verificacion.</p>
+            <Image src={ImageConfirmacion} alt="Imagen Confirmacion" className={Style.Image}/>
+            <Link href='/' className={Style.Link}>Finalizar</Link>
+        </div>
+    </div>
+);
+
+export default Confirmacion;
```
Comment (Israel-Laguan) [inline components/Confirmacion.jsx]:
```suggestion
export default Confirmacion;

```

File: components/SobreNosotros.jsx [hunk 3]
```diff
@@ -4,7 +4,7 @@ import imageSobreNosotros from "../public/sobreNosotros.png";
 import Style from "../styles/SobreNosotros.module.css";
 
 const SobreNosotros = () => (
-    <div className={Style.container_sobrenosotros}>
+    <div id="sobreNosotros" className={Style.container_sobrenosotros}>
```
Comment (Israel-Laguan) [inline components/SobreNosotros.jsx]: The id should be in English

File: styles/Navbar.module.css [hunk 4]
```diff
@@ -1,11 +1,27 @@
...
-  }
+.item {
+  @apply ml-10 text-sm uppercase hover:border-b font-bold;
+}
 
-  .container {
-    @apply fixed w-full h-20 shadow-xl z-[100] bg-[#2BA84A];
-  }
+.container {
+  @apply fixed w-full h-20 shadow-xl z-[100] bg-[#2BA84A];
```
Comment (Israel-Laguan) [inline styles/Navbar.module.css]: Move the bg to tailwind colors

File: styles/Navbar.module.css [hunk 5]
```diff
@@ -1,11 +1,27 @@
...
 
-  .containerNavbar { 
-    @apply flex justify-between items-center w-full h-full px-2 pr-20;
-  }
+.containerNavbar { 
+  @apply flex justify-between items-center w-full h-full px-2 pr-10;
+}
+
+.open {
+  @apply fixed left-0 top-0 w-[100%] sm:w-[60%] md:w-[45%] h-screen bg-[#2f6937] p-10;
```
Comment (Israel-Laguan) [inline styles/Navbar.module.css]: Move the bg to tailwind colors

File: styles/PayForm.module.css [hunk 6]
```diff
@@ -0,0 +1,107 @@
+.ContainerForm {
+    @apply w-4/5 m-auto p-10 flex rounded-lg bg-[darkgreen];
+}
+
+.ContainerMethodPay {
+    @apply w-1/2 border-r-2 border-[#61E176] flex;
```
Comment (Israel-Laguan) [inline styles/PayForm.module.css]: Move the border's color to tailwind colors

File: styles/PayForm.module.css [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+    @apply mt-8;
+}
+
+.ContainerInfo {
+    @apply w-1/2 flex;
+}
+
+.Form {
+    @apply w-screen h-screen flex items-center;
+}
```
Comment (Israel-Laguan) [inline styles/PayForm.module.css]:
```suggestion
}

```

File: components/Marketing.jsx (hunk 1 above)
Comment (Israel-Laguan) [reply components/Marketing.jsx]: Why you changed this? Was good on English

**Review by elias-miguel-vergara — APPROVED** [review]

**Pull request merged**